R/id_ease.R

```diff
@@ -0,0 +1,95 @@
+
```

Comment:
Why is id_ease part of GAMBLR.data?

Reply:
`id_ease` was considered a core GAMBLR function when setting up the structure of GAMBLR child repos, since it calls `get_gambl_metadata`, thus this function is living in the GAMBLR.results repo. Since we don't want users without GSC access to run that package, the helper function needs to be available in this repo as well. At least that's how I rationed. I guess it could, in theory, be moved out from GAMBLR.results and only be in this package (since this is a dependency of GMABLR.results) and then the internal call of `get_gambl_metadata` in `id_ease` would be dictated by what version of `get_gambl_metadata` the user has at hand (i.e the one from GAMBLR.results or the one in this repo).